### DISCUSSION TITLE
File exist but shows no such file or directory

### DISCUSSION THREAD

**Discussion by wooilok123** [termux/termux-packages#7359] [General]
The file is exist but it keeps saying that "no such file or directory"
Can anyone help me out with this? thanks

![WhatsApp Image 2021-08-20 at 12 14 20 AM](https://user-images.githubusercontent.com/68337442/130109159-a32fd92f-703b-4b37-a19f-dbdc7afeae35.jpeg)

**Comment by ghost** [comment]
https://wiki.termux.com/wiki/Differences_from_Linux:
1. [Termux and Android itself are not FHS-compliant](https://wiki.termux.com/wiki/Differences_from_Linux#Termux_is_not_FHS_compliant).
2. [Termux and Android use Bionic libc](https://wiki.termux.com/wiki/Differences_from_Linux#Termux_uses_Bionic_libc).

So if you try to run something that is designed for normal Linux distributions, you probably will experience issues. Fix your scripts, ensure they all have proper shebang:
```
#!/data/data/com.termux/files/usr/bin/sh
```

If have any binary programs, ensure they are compiled for Android using the NDK.
